qa/rpc-tests/txn_doublespend.py

```diff
@@ -44,7 +44,7 @@ def run_test(self):
         # Coins are sent to node1_address
         node1_address = self.nodes[1].getnewaddress("")
 
-        # First: use raw transaction API to send (starting_balance - (mining_reward - 2)) BTC to node1_address,
+        # First: use raw transaction API to send (starting_balance - (mining_reward - 2 ZEC)) to node1_address,
```

Comment:
The parenthesis change is incorrect here IMHO; when read as a sentence, this should be equivalent to "First: use raw transaction API to send some_amount ZEC to node1_address".

If you're already rebasing to fix other issues, squash this in:
```suggestion
        # First: use raw transaction API to send (starting_balance - (mining_reward - 2)) ZEC to node1_address,
```